### DISCUSSION TITLE
[Run issue]:G4Exception: Aborting execution

### DISCUSSION THREAD

**Discussion by Yuri-HubL** [OpenTOPAS/OpenTOPAS#101] [Run-Problem] [answered]
### Contact Details

2633188181@qq.com

### What was the issue?

Dear administrator,

I'm currently installing OpenTOPAS and I'm currently at Step 8 and installed correctly

![image](https://github.com/user-attachments/assets/71ba1bda-0c27-45d7-baec-c7d8f4ee54aa)

After Step 8, I tried to run a sample and got the following error.

![image](https://github.com/user-attachments/assets/edb35a7e-d68f-42ce-a761-6b690c65f9af)

How should I solve this problem?I am looking forward to your reply.

have a nice day,
Yuri

### TOPAS category

Geometry

### Version

4.0

### What Operative System (OS) are you running on?

Windows (WSL)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by tmasilela** [comment]
Hi Yuri,

It looks like TOPAS can't find the G4DATA directory. Please make sure that you didn't make a mistake in Step 9, and you didn't miss the line "export TOPAS_G4_DATA_DIR=~/Applications/GEANT4/G4DATA". Hope this solves your issue.

All the best,
Thongchai

**Reply by Yuri-HubL** [reply]
Hi Thongchai，

I've set up this command and I'm sure my path is correct. But this problem still exists.

![image](https://github.com/user-attachments/assets/58993f41-e19e-421f-b13f-14cb2c93f69d)

best wish,
Yuri

**Reply by Yuri-HubL** [reply]
Hi Thongchai，

I restarted my wsl, and this time it did not report an error saying that the GEATA path was not specified, but another error occurred.

![image](https://github.com/user-attachments/assets/ab6fb374-042f-4233-ae6c-c1f77801a77b)

best wish,
Yuri

**Reply by mgochan** [reply]
Hi Yuri

I noticed I'm having similar issues installing as you did. In step 9, what quotation marks did you use? I think that's where my issue is

**Comment by tmasilela** [comment, marked as answer]
Hi Yuri, 

In terms of your 2nd image my guess is that you're missing the line "export QT_QPA_PLATFORM_PLUGIN_PATH" in your shell script.

In your first image, it seems TOPAS is not finding the G4DATA files because you are using the topas executable from /bin instead of the shellscript you just created. Instead of 

`/home/yuri/Applications/TOPAS/OpenTOPAS-install/bin/topas QtTest.txt`

you should use the following, as specified in Step 10 of the user guide.

`topas QtTest.txt`

Hope that fixes your issues.

All the best,
Thongchai
